### DISCUSSION TITLE
When using proxy-rewrite, is there a solution to get the value of host as regex_uri?

### DISCUSSION THREAD

**Discussion by w7team** [apache/apisix#4982] [General]
我们有个场景，想把对象存储中的多级目录用二级域名的方式转成一级目录。
比如 **http://a.com/a/b/c/d/1.jpg** 转换成 **http://a-b-c-d.b.com/1.jpg** ，所以想问一下如何正则匹配host来实现uri改写

English：
We have a scenario in which we want to convert multi-level directories in the object store into first-level directories by means of second-level domain names. For example, http://a.com/a/b/c/d/1.jpg is converted into http://a-b-c-d.b.com/1.jpg, so I would like to ask how to regularly match host to rewrite URIs.

**Comment by membphis** [comment]
English is better. @w7team

**Comment by tokers** [comment]
Currently it's not feasible to use proxy-rewrite to achieve your demands.
